src/smc-hub/postgres-server-queries.coffee

```diff
@@ -1123,8 +1123,8 @@ exports.extend_PostgreSQL = (ext) -> class PostgreSQL extends ext
             # this includes the email address, except NOT an error if
             # there is no match
             @get_account
-                account_id : if misc.is_valid_uuid_string(opts.query) then opts.query
-                email_address : if misc.is_valid_email_address(opts.query) then opts.query
+                account_id : if misc.is_valid_uuid_string(opts.query) then opts.query.trim()
```

Comment:
This line makes no sense to me, since wouldn't `misc.is_valid_uuid_string(opts.query)` being true imply that the string has not whitespace to trim?

Reply:
Do you really mean to trim `opts.query` before calling these misc.is_valid functions?